### PR TITLE
LPS-144427 | master

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LPS-144427

### CODE REVIEW

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:test:staging

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:staging:nocompile

**Comment by weller0246** [comment]
ci:test:staging:nocompile

File: portal-web/test/functional/com/liferay/portalweb/macros/site/SiteTemplates.macro [hunk 1]
```diff
@@ -406,7 +406,12 @@ definition {
 	macro openSiteTemplateInEditMode {
 		var templateID = SiteTemplates.getSiteTemplateID(siteTemplateName = "${siteTemplateName}");
 
-		Navigator.openWithAppendToBaseURL(urlAppend = "group/${templateID}?p_l_mode=edit");
+		if (isSet(oldestVersion)) {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/macros/site/SiteTemplates.macro:409]: Suggest to replace oldestVersion with widgetPage, it's more readable and straightforward.

**Review by JoyceWang08211 — COMMENTED** [review]
After this change, you can forward to BChan directly. Thx

File: portal-web/test/functional/com/liferay/portalweb/macros/site/SiteTemplates.macro (hunk 1 above)
Comment (weller0246) [reply portal-web/test/functional/com/liferay/portalweb/macros/site/SiteTemplates.macro]: I looked into the test and agree with you, **widgetPage** is clearer, I pushed the changes!

**Comment by JoyceWang08211** [comment]
ci:test:sf

**Comment by JoyceWang08211** [comment]
LGTM

**Comment by JoyceWang08211** [comment]
@weller0246 Please fix the SF error, thx.

**Comment by weller0246** [comment]
@JoyceWang08211 fixed, sorry for the missed sf

**Comment by weller0246** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:staging:nocompile

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM!

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Review by JoyceWang08211 — APPROVED** [review]
LGTM

**Comment by JoyceWang08211** [comment]
ci:forward

**Pull request closed without merging**